scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -76,6 +80,7 @@ namespace <%= namespace %> {
         private IWebCheckout WebCheckout;
         private INativeCheckout NativeCheckout;
         private PaymentSettings StorePaymentSettings;
+        private String ShopName;
```

Comment:
I think calling this `StoreName` would be more fitting. Reason being we want to stay away from `Shopify Shop`